config/webpack.common.js

```diff
@@ -62,6 +62,7 @@ const common = {
       filename: '[name].css',
     }),
   ],
+  target: ['web', 'es2020'],
```

Comment:
Why is the target `web`? Shouldn't it be `node`, since this configuration applies to the library, instead of the built chrome extension?